### ISSUE TITLE
Y24-489 - Show the robot button Bravo LCMT DNA Adp Lig => LCMT DNA Lib PCR after the WGS branch submission

### ISSUE THREAD

**Issue by yoldas** [sanger/limber#2100]
**User story**

As Limber user running LCM Triomics EMSeq and WGS pipeline, I would like the robot button Bravo LCMT DNA Adp Lig => LCMT DNA Lib PCR to be available **after** the WGS branch submission is done.

"After marking the LCMT DNA Adp Lig plate as passed using Manual Transfer, upon scanning the plate back into Limber the ‘WGS Branch – Automated Submission’ button becomes available as expected. However, the bed verification step ‘Bravo LCMT DNA Adp Lig => LCMT DNA Lib PCR’ is already exposed to the user before the submission has been made, which is potentially slightly confusing because this step is only required if the WGS branch is required (as the LCMT DNA Lib PCR plate does not exist before the button is clicked). However I do appreciate that it is also the case that the user only needs to scan the passed LCMT DNA Adp Lig plate back into Limber if the WGS branch is needed; otherwise the LCMT TET2 Ox plate becomes the next point of interaction with Limber and none of the above will be seen. I’m happy with the current arrangement, but if there is some simple way to ensure that the bed verification option is only visible after the WGS branch has been generated that would be useful."

**Who are the primary contacts for this story**
Scott G.
Abdullah

**Who is the nominated tester for UAT**
Scott G.

**Acceptance criteria**

To be considered successful the solution must allow:
- [x] The robot button Bravo LCMT DNA Adp Lig => LCMT DNA Lib PCR is shown after WGS branch is submitted.
- [x] The robot button respects the pipeline filters, similar to suggested child purposes.
- [x] The robot button is shown after the LCMT DNA Lib PCR plate is created.
- [x] Integration suite is updated to check the availability of the button.

**Dependencies**

This story is blocked by the following dependencies:
N/A

References
This story has a non-blocking relationship with:
- #2065

**Additional context**
This may be implemented as a generic feature to make sure robot buttons are not rendered when they are not relevant. However, it aims to solve a specific UX issue of the LCM Triomics EMSeq and WGS pipeline.

The following screenshot shows that the Bravo LCMT DNA Adp Lig => LCMT DNA Lib PCR button is available even though WGS Branch - Automated Submissions is not done and the LCMT DNA Lib PCR plate is not added yet.

![Screenshot 2024-12-02 at 15 45 41](https://github.com/user-attachments/assets/a9250ebb-0c9f-4f31-a0af-d913bc989042)

**Comment by TWJW-SANGER** [comment]
Message from Scot G below - the upshot is, I think, we should prioritise the forthcoming LCMT -> RNASeq integration stories over this.
----
Hi Tom,
This is a shame, however as I mentioned in the original story this was very much a nice-to-have feature so if implementing it has caused other pipelines to break then I’m tempted to say that you should just consider it a ‘won’t have’ rather than a ‘could have’. I think we can live with the current arrangement without any real difficulty, and I’d rather any future effort on the LCMT pipeline was spent on the RNAseq integration stories we are working up with Stanley as opposed to this story.
